public/stylesheets/adminLogin.css

```diff
@@ -1,5 +1,4 @@
 body {
-    padding-top: 40px;
```

Comment:
Removing this padding fixes that extra space above the navbar on some pages that didn't appear on others. (The NavBar is the first element in the "body").